wman-backend/Wman.Logic/Classes/UserLogic.cs

```diff
@@ -36,6 +36,10 @@ public async Task<IEnumerable<WorkloadDTO>> GetWorkLoads(IEnumerable<string> use
                 {
                     throw new ArgumentException(String.Format("User: {0} doesn't exists!", username));
                 }
+                if (await userManager.IsInRoleAsync(selectedUser, "Worker") == false)
```

Comment:
Same advice applies here too.
```suggestion
                if (!await userManager.IsInRoleAsync(selectedUser, "Worker"))
```